lib/process-services-cloud/src/lib/form/components/widgets/display-rich-text/display-rich-text.widget.spec.ts

```diff
@@ -18,7 +18,7 @@
 import { DebugElement } from '@angular/core';
 import { ComponentFixture, TestBed } from '@angular/core/testing';
 import { By } from '@angular/platform-browser';
-import { CoreTestingModule, setupTestBed } from '../../../../testing';
+import { CoreTestingModule, setupTestBed } from '../../../../../../../core/src/lib/testing';
```

Comment:
I think we need a ProcessServiceCloudTestingModule instead of the CoreTestingModule